### PR TITLE
Add grey 900

### PR DESCRIPTION
**Pull request by CCAyl**
- adds grey 900 to core colours
- update `primary.background.action` to grey 900
figma: https://www.figma.com/file/784YociOk102q8PgCz0xAl?node-id=1226:9738&mode=design#492085321

### CODE REVIEW

**Review by alexandra-lim — COMMENTED** [review]
Looking good. Are you planning on updating primary.borderActive to also be grey900?

Are the updates from 600 to 700, etc. going to be done in a follow up MR?

**Comment by CCAyl** [comment]
> Looking good. Are you planning on updating primary.borderActive to also be grey900?
> 
> Are the updates from 600 to 700, etc. going to be done in a follow up MR?

according to the tokens file borderActive is still 800?

what should be updated from 600 to 700?

**Comment by CCAyl** [comment]
ah gotcha, make sense, added!

File: src/data/light.json [hunk 1]
```diff
@@ -203,12 +203,12 @@
...
           "type": "color",
-          "description": "Used for t4e border colour for primary components such as buttons and Message"
+          "description": "Used for the border colour for primary components such as buttons and Message"
         },
         "hover": {
-          "value": "{colors.grey.500}",
+          "value": "{colors.grey.800}",
           "type": "color",
           "description": "Used for the hovered border colour for primary components such as buttons"
         },
```
Comment (alexandra-lim) [inline src/data/light.json:214]: Maybe I'm using Github wrong but it won't let me comment on code that hasn't been changed.

Let's update the primary.borderActive to be 900 too

File: src/data/light.json
Comment (alexandra-lim) [inline src/data/light.json:1]: Let's update `primary.text` to be white too!

**Comment by CCAyl** [comment]
hopefully i've added everything now!

**Review by alexandra-lim — APPROVED** [review]
Nice! Thank you!

**Pull request merged**